relays/bin-substrate/src/cli/encode_call.rs

```diff
@@ -246,4 +246,72 @@ mod tests {
 			"0x0d00d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27de5c0"
 		);
 	}
+
+	#[test]
+	fn should_encode_remark_with_default_payload() {
+		// given
+		let mut encode_call = EncodeCall::from_iter(vec!["encode-call", "RialtoToMillau", "remark"]);
+
+		// when
+		let hex = encode_call.encode().unwrap();
+
+		// then
+		assert!(format!("{:?}", hex).starts_with("0x080154556e69782074696d653a"));
```

Comment:
Default payload is using current time, right? So this test will always fail

Reply:
Note that this one is not exact, but only checks the beginning (`.starts_with`). The timestamp comes after some prefix, so we only check the prefix.